.github/workflows/cli_checks.yml

```diff
@@ -16,19 +17,6 @@ env:
   DEFAULT_PYTHON_VERSION: "3.10.7"
 
 jobs:
-  Test-Envs:
-    runs-on: ubuntu-latest
-    timeout-minutes: 25
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-
-      - name: Install Nox
-        run: pip install nox>=2022
-
-      - name: Initialize the test environment
-        run: nox -s "fides_env(test)" -- test
-
```

Comment:
@ThomasLaPiana: this is exercised by the `cypress_e2e.yml` checks now, which does even more than just ensuring the test env comes up: it runs a privacy request 👍